common/flatpak-transaction.c

```diff
@@ -2879,6 +2900,89 @@ flatpak_transaction_run (FlatpakTransaction *transaction,
   return FLATPAK_TRANSACTION_GET_CLASS (transaction)->run (transaction, cancellable, error);
 }
 
+static gboolean
+get_installation_dir_free_space (FlatpakInstallation  *installation,
+                                 guint64              *free_space,
```

Comment:
Add `_bytes` to make the units more obvious:

```suggestion
                                 guint64              *free_space_bytes,
```